create-application.md

```diff
@@ -0,0 +1,121 @@
+
+```yaml
+name: candy
+instance-type: a2.3
+image: default
+boot-activity: com.canonical.candy.GameApp
+required-permissions:
+  - android.permission.WRITE_EXTERNAL_STORAGE
+  - android.permission.READ_EXTERNAL_STORAGE
+addons:
+  - ssh
+tags:
+  - game
+extra-data:
+  com.canonical.candy.obb:
+    target: /data/app/com.canonical.candy-1/lib
+  game-data-folder:
+    target: /sdcard/Android/data/com.canonical.candy/
+watchdog:
+  disabled: false
+  allowed-packages:
+    - com.android.settings
+services:
+  - name: adb
+    port: 5559
+    protocols: [tcp]
+    expose: false
+resources:
+  memory: 4GB
+  disk-size: 8GB
+```
+
+See [Application manifest](tbd) for detailed information about all available attributes.
+
+## Create from a directory
+
+When creating an application from a directory, the directory should contain the required components for the creation:
+
+* `manifest.yaml`
+* `app.apk`
+* `extra-data` (optional)
+
+> **Note:** Due to Snap strict confinement, the directory must be located in the home directory.
+
+With everything in place, create the application by entering the following command:
+
+    amc application create <path/to/application-content>
+
+When the `create` command returns, the application package is uploaded to the AMS service and the [bootstrap process](tbd) is started. The application is not yet ready to be used. You can watch the status of the application with the following command:
+
+    amc application show bcmap7u5nof07arqa2ag
+
+The returned output looks similar to the following:
+
+```bash
+id: bcmap7u5nof07arqa2ag
+name: candy
+status: initializing
+published: false
+config:
+  instance-type: a2.3
+  boot-package: com.canonical.candy
+versions:
+  0:
+    image: bf7u4cqkv5sg5jd5b2k0 (version 0)
+    published: false
+    status: initializing
+    addons:
+    - ssh
+    boot-activity: com.canonical.candy.GameApp
+    required-permissions:
+    - android.permission.WRITE_EXTERNAL_STORAGE
+    - android.permission.READ_EXTERNAL_STORAGE
+    extra-data:
+      com.canonical.candy.obb:
+        target: /data/app/com.canonical.candy-1/lib
+      game-data-folder:
+        target: /sdcard/Android/data/com.canonical.candy/
+    watchdog:
+      disabled: false
+      allowed-packages:
+      - com.android.settings
+    services:
+    - port: 5559
+      protocols:
+      - tcp
+      expose: false
+      name: adb
+resources:
+  memory: 4GB
+  disk-size: 8GB
+```
+
+Once the status of the application switches to `ready`, the application is ready and can be used. See [Wait for an application](tbd) for information about how to monitor the application status.
+
+## Create from a tarball
+
+An application can also be created from a tarball file. The tarball file must be compressed with bzip2 and must follow the format of the application package as described before.
```

Comment:
What do we mean with "as described before"? The folder structure from line 50-52?

Reply:
Yes